src/Controls/src/Core/HandlerImpl/View.Impl.cs

```diff
@@ -26,5 +29,22 @@ private protected override void OnHandlerChangingCore(HandlerChangingEventArgs a
 
 			base.OnHandlerChangingCore(args);
 		}
+
+
+#if __ANDROID__
+		void IAccessibilityNodeInfoListener.InitializeAccessibilityNodeInfo(AndroidX.Core.View.Accessibility.AccessibilityNodeInfoCompat? nodeInfo)
```

Comment:
The only issue that might come up is if I create a custom view and then also implement this interface. I will be replacing the functionality of this method and have no way to get to it.

Reply:
This might also be something that is worth adding to a test. What will a derived view look like if it also needs to set this.

Reply:
Yea, I'm not too sure how useful it is to retain our behavior if a user chooses to influence the node info. This is one area where the mapper approach was slightly better because if you wanted to change the behavior then it's a known paradigm you are hooking into.  

For discussion purposes, I moved this code out to a public extension method that users can call and exposed a platform specific override.